### ISSUE TITLE
PHP error occurred

### ISSUE THREAD

**Issue by khaiphan9x** [hollodotme/fast-cgi-client#54]
```
Deprecated: Array and string offset access syntax with curly braces is deprecated in /vendor/hollodotme/fast-cgi-client/src/Encoders/PacketEncoder.php on line 57
```
This is my PHP code
```
require '../vendor/autoload.php';

use hollodotme\FastCGI\Client;
use hollodotme\FastCGI\Requests\PostRequest;
use hollodotme\FastCGI\SocketConnections\UnixDomainSocket;
use hollodotme\FastCGI\RequestContents\MultipartFormData;



$connection = new UnixDomainSocket('php.sock');
$client = new Client();

$formData = [
			'testKey1' => 'value1',
			'testKey2' => 'value2',
		];
$files = [
			'testFile1' => '/TestFile.txt',
			'testFile2' => '/TestFile.txt',
		];

$multipartFormData = new MultipartFormData($formData, $files);
$postRequest = new PostRequest(
	'/test.php',
	$multipartFormData->getContent()
);
$postRequest->setContentType($multipartFormData->getContentType());

$response = $client->sendRequest($connection, $postRequest);

print_r($response);
```

## Specifications

- PHP version: 7.4.3

**Comment by hollodotme** [comment]
Hi, you probably have the wrong version installed. This issue should be fixed already in v3.0.1

**Comment by khaiphan9x** [comment]
> Hi, you probably have the wrong version installed. This issue should be fixed already in v3.0.1

Because $var{foo} was removed in PHP7.4x, instead of $var[foo]

**Comment by khaiphan9x** [comment]
> Hi, you probably have the wrong version installed. This issue should be fixed already in v3.0.1
Is this package a proxy between the user (browser) and php-fpm?  I am looking for a similar package for the project, I tried to edit the fast-cgi-client that is suitable for that.  Thanks.
https://github.com/hollodotme/fast-cgi-proxy
